WordPress/Classes/ViewRelated/Me/App Settings/AppSettingsViewController.swift

```diff
@@ -475,58 +485,84 @@ private extension AppSettingsViewController {
         )
     }
 
-    func otherTableSection() -> ImmuTableSection {
-        let otherHeader = NSLocalizedString("Other", comment: "Link to About section (contains info about the app)")
+    func appearanceTableSection() -> ImmuTableSection {
+        var rows: [ImmuTableRow] = []
 
-        let debugRow = NavigationItemRow(
-            title: NSLocalizedString("Debug", comment: "Navigates to debug menu only available in development builds"),
-            icon: .gridicon(.bug),
-            action: pushDebugMenu()
+        let colorModeRow = NavigationItemRow(
+            title: NSLocalizedString("Color Mode", comment: "The title of the app color mode settings screen"),
+            detail: AppAppearance.current.appearanceDescription,
+            action: pushColorModeSettings()
         )
+        rows.append(colorModeRow)
 
-        let iconRow = NavigationItemRow(
-            title: NSLocalizedString("App Icon", comment: "Navigates to picker screen to change the app's icon"),
-            action: pushAppIconSwitcher()
+        if AppConfiguration.allowsCustomAppIcons && UIApplication.shared.supportsAlternateIcons {
+            let appIconRow = NavigationItemRow(
+                title: NSLocalizedString("App Icon", comment: "Navigates to picker screen to change the app's icon"),
+                action: pushAppIconSwitcher()
+            )
+            rows.append(appIconRow)
+        }
+
+        let appColorRow = NavigationItemRow(
+            title: accentColorTitle,
+            detail: AppColor.accent.description,
+            action: pushAppColorList()
+        )
+        rows.append(appColorRow)
+
+        return ImmuTableSection(
+            headerText: NSLocalizedString("Appearance", comment: "The title of the app appearance settings section"),
+            rows: rows,
+            footerText: nil
         )
+    }
+
+    var accentColorTitle: String {
+        NSLocalizedString("Accent Color", comment: "The title of the app accent color")
```

Comment:
For new strings, we include a key with them. Same thing applies for the other new strings but I don't want to spam. For more info, we have a localization guide [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/docs/localization.md#use-unique-reverse-dns-naming-style-keys).

You don't have to worry about updating strings which already exist though.

Reply:
Oh, I guess I was trying to keep consistency with other localized strings in this file, but I understand & will fix this for the new strings.